tracer/src/Datadog.Trace/Propagators/W3CContextPropagator.cs

```diff
@@ -17,6 +17,8 @@ internal class W3CContextPropagator : IContextInjector, IContextExtractor
         /// </summary>
         public const string TraceParent = "traceparent";
 
+        public static readonly W3CContextPropagator Instance = new();
```

Comment:
Can we rename this class to `W3CTraceContextPropagator`?

Reply:
Done!

I had thought of renaming this file, but the others follow a `{Name}ContextPropagator` convention and I didn't want to make this one `W3CTraceContextContextPropagator`. Your suggestion works for me so I did that instead.